src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Insights/Add-ArmApplicationInsightsMetricAlertWebTestCriteria.ps1

```diff
@@ -0,0 +1,37 @@
+function Add-ArmApplicationInsightsMetricAlertWebTestCriteria {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationInsightsMetricAlert")]
+    Param(
+        [PSTypeName("ApplicationInsightsMetricAlert")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $MetricAlert,
+        [PSTypeName("ApplicationInsights")]
+        [Parameter(Mandatory)]
+        $ApplicationInsights,
+        [PSTypeName("ApplicationInsightsWebTest")]
+        [Parameter(Mandatory)]
+        $WebTest,
```

Comment:
Can we get ApplicationInsights object from WebTest?

Reply:
We could get it from "dependsOn", but I don't think there's any way to easily determine if id is actually AppInsights